lib/puppet/provider/foreman_config_entry/cli.rb

```diff
@@ -52,11 +52,11 @@ def self.prefetch(resources)
     return if entries.nil?
     resources.each do |name, resource|
       provider = entries.find { |entry| entry.name == name }
-      if provider.nil? && resource[:ignore_missing]
-        # just consider it has a value we exepcted
+      if provider.nil? and resource[:ignore_missing]
```

Comment:
Ugh, you're too quick for me. I'd already changed that
On Mar 30, 2016 9:17 AM, "Dominic Cleal" <notifications@github.com> wrote:

> In lib/puppet/provider/foreman_config_entry/cli.rb
> <https://github.com/theforeman/puppet-foreman/pull/420#discussion_r57845569>
> :
>
> > @@ -52,11 +52,11 @@ def self.prefetch(resources)
> >      return if entries.nil?
> >      resources.each do |name, resource|
> >        provider = entries.find { |entry| entry.name == name }
> > -      if provider.nil? && resource[:ignore_missing]
> > -        # just consider it has a value we exepcted
> > +      if provider.nil? and resource[:ignore_missing]
>
> && is preferred in expressions, can you restore the original line please?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/puppet-foreman/pull/420/files/4cd3a71c0311508f33be3159ddf1ded54c1540df#r57845569>
>